frontend/src/components/classification-dropdown.js

```diff
@@ -5,7 +5,7 @@ import {
   Dropdown,
   DropdownItem,
   DropdownToggle
-} from '@patternfly/react-core';
+} from '@patternfly/react-core/deprecated';
```

Comment:
Why are we pulling from deprecated here?

Reply:
@mshriver fixed